src/blueprints/Do.Blueprints.Service.Application/Configuration/EnvironmentExtensions.cs

```diff
@@ -1,20 +1,20 @@
-using Do.Architecture;
+using Do.Database;
 using Microsoft.Extensions.Hosting;
 
 namespace Do;
 
 public static class EnvironmentExtensions
 {
-    public static IFeature ForDevelopment(this IFeature @default, IFeature featureOnDevelopment) =>
+    public static IDatabaseFeature ForDevelopment(this IDatabaseFeature @default, IDatabaseFeature featureOnDevelopment) =>
         @default.For(Environments.Development, featureOnDevelopment);
 
-    public static IFeature ForStaging(this IFeature @default, IFeature featureOnStaging) =>
+    public static IDatabaseFeature ForStaging(this IDatabaseFeature @default, IDatabaseFeature featureOnStaging) =>
         @default.For(Environments.Staging, featureOnStaging);
 
-    public static IFeature ForProduction(this IFeature @default, IFeature featureOnProduction) =>
+    public static IDatabaseFeature ForProduction(this IDatabaseFeature @default, IDatabaseFeature featureOnProduction) =>
         @default.For(Environments.Production, featureOnProduction);
 
-    public static IFeature For(this IFeature @default, string environment, IFeature featureOnEnvironment)
+    public static IDatabaseFeature For(this IDatabaseFeature @default, string environment, IDatabaseFeature featureOnEnvironment)
```

Comment:
these should be generic, not specific to `IDatabaseFeature`

`TFeature ForProduction<TFeature>(this TFeature @default, TFeature featureOnProduction) where TFeature : IFeature`